dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentActionlet.java

```diff
@@ -0,0 +1,87 @@
+package com.dotmarketing.portlets.workflows.actionlet;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.workflows.model.*;
+import com.dotmarketing.util.Logger;
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Do the save for a content (checkin)
+ * @author jsanca
+ */
+public class SaveContentActionlet extends WorkFlowActionlet {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)